### ISSUE TITLE
Strange movement while gravity is set to Up.

### ISSUE THREAD

**Issue by Limesar** [Gaider10/GravityChanger#25]
While the gravity is set to Up, movement while facing directly east/west is swapped (i.e. pressing W to moves you backward relative to your perspective instead of forward). Facing between north/south and east/west also alters movement, and changing the config file has no effect on this ("keepWorldLook", "client", and "server" have all been changed to no avail).

My current version of each required mod includes:
GravityChanger-0.3.0-1.18.1
cloth-config-6.1.48-fabric
fabric-api-0.44.0+1.18
fabric-loader-0.12.11-1.18.1

By the looks of it, #19 also mentions this. Something must have happened with the latest update that messed with this fix. I'm not that great at coding, so I likely won't be much help in that regard.

**Comment by Gaider10** [comment]
Are you using any other mods? I tested the mod before releasing and did not have any problems, so it is probably an incompatibility with one of your other mods

**Comment by Limesar** [comment]
The only two mods I could think of that might've caused compatibility issues was Pehkui and a sitting mod (for slabs/stairs), but when I deleted the GravityChanger and put it back in my mods folder to test without those other mods, my movements were fixed and became relative to my orientation. Putting back the other two mods didn't change this.

What I found odd about this was, when I was having the issue, I had both mods installed, but removing Pehkui didn't change the issue I was having.

Next, I tried changing keepWorldLook to true, then to false, then back to true. My issue never returned.

I then realized that, between deleting GravityChanger and writing this post, I switched from Optifine (using OptiFabric) to Sodium w/ Iris. It turns out _that_ was the reason I had this issue in the first place.

Had I not checked this thread and checked my mods at your suggestion, I probably wouldn't have found this out. I guess the real issue is that Optifine/OptiFabric isn't compatible with this mod, but I can live with that.
